sqlMesh/models/intermediate/wdi.py

```diff
@@ -29,45 +37,32 @@ def execute(
     execution_time: datetime,
     **kwargs: t.Any,
 ) -> pd.DataFrame:
-    
-    # connect ibis to database
-    con = ibis.duckdb.connect(DB_PATH)
+    """Process WDI data and return the transformed Ibis table.
 
-    """Process WDI data and return the transformed Ibis table."""
+    Args:
+        context: SQLMesh execution context
+        start: Start date for data processing
+        end: End date for data processing
+        execution_time: Timestamp of execution
+        **kwargs: Additional keyword arguments
 
+    Returns:
+        Processed WDI data as a pandas DataFrame
+    """
     print("Starting wdi_data")
     wdi_table = context.table("wdi.csv")
-    wdi_df = context.fetchdf(f"SELECT * FROM {wdi_table}")
-    wdi_data = (ibis.memtable(wdi_df, name='wdi')
+    wdi_df = context.fetchdf(wdi_table.select("*"))  # type: ignore
```

Comment:
This file seems to have a few changes unrelated to PR description. Just want to make sure those are intentional and were also tested even though they look good.